pkgs/os-specific/linux/iptables/default.nix

```diff
@@ -1,19 +1,35 @@
 { lib, stdenv, fetchurl, pkg-config, pruneLibtoolFiles, flex, bison
 , libmnl, libnetfilter_conntrack, libnfnetlink, libnftnl, libpcap
 , nftablesCompat ? true
+, fetchpatch
 }:
 
 with lib;
```

Comment:
It would be nice to remove this.

Reply:
fetchpatch is used

Reply:
I was talking about the `with lib` line.

Reply:
Oh, sorry. Fixed.